packages/pn-commons/src/utils/date.utility.ts

```diff
@@ -66,7 +66,7 @@ export function isToday(date: DatePickerTypes): boolean {
   );
 }
 
-export function formatDate(dateString: string): string {
+export function formatDate(dateString: string, todayLabelizzation: boolean = true): string {
```

Comment:
Add test for the case toadyLabelizzation = false (check if case true exists, otherwise add the test)